### PR TITLE
Delete Contact Feature File

### PR DESCRIPTION
**Pull request by Samikshap85**
### issues
https://github.com/SagarGi/FeatureFiles/issues/5

### CODE REVIEW

File: Features1/tests/features/deleteContact.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature:Delete Contact
+    As a user
+    I want to delete my contacts 
+    So that i can remove unwanted contacts
+
+    Scenario: Delete contacts
+        Given the user has navigated to the contact page
+        When  the user click "delete" botton
```
Comment (SagarGi) [inline Features1/tests/features/deleteContact.feature]: how would you know which `delete` button to click if there are multiple contacts? may be give user information which one to delete.

File: Features1/tests/features/deleteContact.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        Then  the contact should be deleted
```
Comment (SagarGi) [inline Features1/tests/features/deleteContact.feature]: and assert that the given user should be deleted. (the user should not be in the contact list anymore)

File: Features1/tests/features/deleteContact.feature [hunk 3]
```diff
@@ -5,5 +5,11 @@ Feature:Delete Contact
 
     Scenario: Delete contacts
         Given the user has navigated to the contact page
-        When  the user click "delete" botton
-        Then  the contact should be deleted
+        And have the following contact list
+            |name    |email             |contact-type|
+            |samiksha|samiksha@gmail.com|Personal    |
+            |anusha  |anusha@gmail.com  |Professional|
```
Comment (kiranadh1452) [inline Features1/tests/features/deleteContact.feature]:
```suggestion
        And the user has added the following contacts
            |name    |email             |contact-type|
            |samiksha|samiksha@gmail.com|Personal    |
            |anusha  |anusha@gmail.com  |Professional|
```

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features1/tests/features/deleteContact.feature [hunk 4]
```diff
@@ -4,16 +4,22 @@ Feature:Delete Contact
     So that i can remove unwanted contacts
 
     Scenario: Delete specific user from contact list
-        Given the user has navigated to the contact page
+        Given the users has navigated to the contact page
         And the following user has been added to the contact list
```
Comment (kiranadh1452) [inline Features1/tests/features/deleteContact.feature]:
```suggestion
        Given the user has navigated to the contact page
        And the following users have been added to the contact list
```

File: Features1/tests/features/deleteContact.feature [hunk 5]
```diff
@@ -4,16 +4,22 @@ Feature:Delete Contact
...
             |name    |email             |contact-type|
             |samiksha|samiksha@gmail.com|Personal    |
             |anusha  |anusha@gmail.com  |Professional|
+            |binita  |binita@gmail.com  |Professional|
+            |susma   |susma@gmail.com   |Personal    |
         When the user deletes following contact with email
             |email             |
             |samiksha@gmail.com|
         Then the  user with email "samiksha@gmail.com" should not  be in contact list anymore
```
Comment (kiranadh1452) [inline Features1/tests/features/deleteContact.feature]:
```suggestion
        Then the user with email "samiksha@gmail.com" should not  be in contact list anymore
```

Comment (kiranadh1452) [inline Features1/tests/features/deleteContact.feature]:
```suggestion
        Then the  user with email "samiksha@gmail.com" should not be in contact list anymore
```

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features1/tests/features/deleteContact.feature [hunk 6]
```diff
@@ -0,0 +1,33 @@
+Feature:Delete Contact
+    As a user
+    I want to delete my contacts 
+    So that i can remove unwanted contacts
+
+    Background:
+        Given the user has navigated to the contact page
+
+
```
Comment (SagarGi) [inline Features1/tests/features/deleteContact.feature]:
```suggestion
```

File: Features1/tests/features/deleteContact.feature [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    Scenario: Delete specific user from contact list
+        Given the following contacts have been added to the contact list
+                |Name    |Email              |Phone    |Contact Type|
+                |Samiksha|Samiksha@gmail.com |9846000  |Personal    |
+                |Binita  |Binita@gmail.com   |984600110|Personal    |
+                |Anusha  |Anusha@gmail.com   |9846000  |Personal    |
```
Comment (SagarGi) [inline Features1/tests/features/deleteContact.feature]:
```suggestion
                |Name    |Email              |Phone    |Contact Type|
                |Samiksha|Samiksha@gmail.com |9846000  |Personal    |
                |Binita  |Binita@gmail.com   |984600110|Personal    |
                |Anusha  |Anusha@gmail.com   |9846000  |Personal    |
```

make its intendation same as below

**Review by SagarGi — APPROVED** [review]

**Pull request merged**